tests/components/androidtv/patchers.py

```diff
@@ -153,6 +153,7 @@ def isfile(filepath):
 
 PATCH_ISFILE = patch("os.path.isfile", isfile)
 PATCH_ACCESS = patch("os.access", return_value=True)
+PATCH_GET_HOST_IP = patch("socket.gethostbyname", return_value="127.0.0.1")
```

Comment:
Ideally we patch only the location of one we want to mock and not the global one

Reply:
I do not exactly understand what you mean here.

Reply:
He wants to see socket patched as it is imported